controllers/storagecluster/cephcluster.go

```diff
@@ -503,6 +503,10 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 		cephCluster.Spec.DisruptionManagement.OSDMaintenanceTimeout = sc.Spec.ManagedResources.CephCluster.OsdMaintenanceTimeout
 	}
 
+	if sc.Spec.ManagedResources.CephCluster.HealthCheck != nil {
+		cephCluster.Spec.HealthCheck =  *sc.Spec.ManagedResources.CephCluster.HealthCheck
```

Comment:
nit: extra space, though I'm curious why the CI didn't catch it with `go vet`
```suggestion
		cephCluster.Spec.HealthCheck = *sc.Spec.ManagedResources.CephCluster.HealthCheck
```

Reply:
done 
```
$ go fmt /home/oviner/DEV_REPOS/ocs-operator/controllers/storagecluster/cephcluster.go
controllers/storagecluster/cephcluster.go

```